servlet/src/main/scala/org/http4s/servlet/syntax/ServletContextSyntax.scala

```diff
@@ -88,4 +88,24 @@ final class ServletContextOps private[syntax] (val self: ServletContext) extends
   }
 }
 
+object ServletContextOps {
+
+  @deprecated("Preserved for binary compatibility", "0.23.11")
+  def `mountHttpApp$extension`[F[_]: Async](
```

Comment:
We could consider making this `private[syntax]`. It would break bincompat for the `static` forwarder, but usually these are only used by Java users (not other Scala users).

Reply:
Actually, we only need this for 2.12.